### PR TITLE
doc: qemu: clarify how to emulate multiple devices

### PR DESCRIPTION
**Pull request by camilamacedo86**
Signed-off-by: Camila Macedo <camila.macedo@foundries.io>

## Readiness

* [x ] Merge (pending reviews)
* [ ] Merge after _date or event_
* [ ] Draft

## Overview

_Why merge this PR? What does it solve?_

## Checklist

_Optional. Add a 'x' to steps taken._
_You can fill this out after opening the PR. "Did I..."_

* [ ] Run spelling and grammar check, preferably with linter.
* [ ] Avoid changing any header associated with a link/reference.
* [ ] Step through instructions (or ask someone to do so).
* [ ] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
* [ ] Match tone and style of page/section.
* [ ] Run `make linkcheck`.
* [ ] View HTML in a browser to check rendering.
* [ ] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
* [ ] follow best practices for commits.
  * [ ] Descriptive title written in the imperative.
  * [ ] Include brief overview of QA steps taken.
  * [ ] Mention any related issues numbers.
  * [ ] End message with sign off/DCO line (`-s, --signoff`).
  * [ ] Sign commit with your gpg key (`-S, --gpg-sign`).
  * [ ] Squash commits if needed.
* [ ] Request PR review by a technical writer and at least one peer.

## Comments

_Any thing else that a maintainer/reviewer should know._
_This could include potential issues, rational for approach, etc._

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 33ab1ac7ae1f62410ba430ac421703b64060e6a2 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2436/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for 787d58b84e061b3a9495529b0402a2297d7d3161 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2437/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for fa922b71a765d78eae67a80b680942ad78e8e35c are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2438/docs/artifacts/html/index.html

File: source/getting-started/emulation-with-qemu/index.rst [hunk 1]
```diff
@@ -113,6 +113,11 @@ Once you’ve converted and resized the image, you can then use it with your QEM
    For this specific platform, we use the ``-bios=flash.bin`` flag to boot the system.
    However, the flags and configurations may vary based on the selected platform.
 
+.. note::
+
+   If you're looking to emulate multiple devices, ensure you convert to the QCOW2 format.
```
Comment (kprosise) [inline source/getting-started/emulation-with-qemu/index.rst:118]:
```suggestion
   If you wish to emulate multiple devices, ensure that you convert to the QCOW2 format.
```

File: source/reference-manual/qemu/qemu-instructions.template [hunk 2]
```diff
@@ -13,8 +13,8 @@
 
 .. note::
     You can fetch either the compressed ``.wic.gz`` or the ``.qcow2`` artifact, you do not need both.
-    Use the .qcow2 artifact if you wish to change the QEMU disk size.
-
+    Use the .qcow2 artifact if you wish to change the QEMU disk size or If you're looking to emulate multiple devices.
```
Comment (kprosise) [inline source/reference-manual/qemu/qemu-instructions.template:16]:
```suggestion
    Use the .qcow2 artifact if you wish to change the QEMU disk size or If you are looking to emulate multiple devices.
```

**Review by kprosise — APPROVED** [review]
A couple suggestions, otherwise it LGTM.

File: source/getting-started/emulation-with-qemu/index.rst [hunk 3]
```diff
@@ -113,6 +113,11 @@ Once you’ve converted and resized the image, you can then use it with your QEM
    For this specific platform, we use the ``-bios=flash.bin`` flag to boot the system.
    However, the flags and configurations may vary based on the selected platform.
 
+.. note::
+
+   If you wish to emulate multiple devices, ensure that you convert to the QCOW2 format.
+   Each image that is converted and subsequently run with QEMU will be recognized as a distinct device.
```
Comment (angolini) [inline source/getting-started/emulation-with-qemu/index.rst:119]:
```suggestion
   To emulate multiple devices, ensure that you convert to the QCOW2 format.
   Each image converted and subsequently run with QEMU will be recognized as a distinct device.
```

Personally, I avoid using "wish" in documentation. I suggest a straightforward way to say the same.

File: source/reference-manual/qemu/qemu-instructions.template [hunk 4]
```diff
@@ -13,8 +13,8 @@
 
 .. note::
     You can fetch either the compressed ``.wic.gz`` or the ``.qcow2`` artifact, you do not need both.
-    Use the .qcow2 artifact if you wish to change the QEMU disk size.
-
+    Use the .qcow2 artifact if you wish to change the QEMU disk size or If you are looking to emulate multiple devices.
+    Each image that is converted and subsequently run with QEMU will be recognized as a distinct device.
```
Comment (angolini) [inline source/reference-manual/qemu/qemu-instructions.template:17]:
```suggestion
    Each image converted and subsequently run with QEMU will be recognized as a distinct device.
```

**Review by angolini — APPROVED** [review]
LGTM

few minor suggestion

File: source/reference-manual/qemu/qemu-instructions.template [hunk 5]
```diff
@@ -13,8 +13,8 @@
 
 .. note::
     You can fetch either the compressed ``.wic.gz`` or the ``.qcow2`` artifact, you do not need both.
-    Use the .qcow2 artifact if you wish to change the QEMU disk size.
-
+    Use the .qcow2 artifact if you wish to change the QEMU disk size or If you are looking to emulate multiple devices.
```
Comment (kprosise) [inline source/reference-manual/qemu/qemu-instructions.template:16]:
```suggestion
    Use the .qcow2 artifact if you wish to change the QEMU disk size, or if you are looking to emulate multiple devices.
```

**Comment by doanac** [comment]
Docs for a4e6df085d25fa6765af364bea712fd3e9d85057 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2445/docs/artifacts/html/index.html

File: source/reference-manual/qemu/qemu-instructions.template (hunk 5 above)
Comment (kprosise) [reply source/reference-manual/qemu/qemu-instructions.template:16]: @camilamacedo86, one last suggestion, then I would be happy to go ahead and merge!

**Comment by doanac** [comment]
Docs for 4344cfe3afd98386e460a2410fda94d9a30213a1 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2446/docs/artifacts/html/index.html

**Pull request merged**